client/settings/connect-stripe-account/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import { __ } from '@wordpress/i18n';
+import { Button, Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+import StripeBanner from '../../payment-method-icons/stripe-banner';
+
+const CardWrapper = styled( Card )`
+	max-width: 560px;
+
+	h2 {
+		font-size: 16px;
+	}
+
+	img {
+		width: 100%;
+	}
+`;
+
+const InformationText = styled.p`
+	color: #1e1e1e;
+`;
+
+const TermOfServicesText = styled.p`
```

Comment:
This should probably be:

```suggestion
const TermsOfServiceText = styled.p`
```